src/coreclr/jit/emitxarch.cpp

```diff
@@ -662,6 +662,120 @@ bool emitter::emitIsInstrWritingToReg(instrDesc* id, regNumber reg)
         case INS_call:
             return true;
 
+        case INS_imul_AX:
+            if (reg == REG_RAX)
+            {
+                return true;
+            }
+            break;
```

Comment:
nit: Group these all as one case and use `if (reg == inst3opImulReg(ins))` instead

Reply:
done